pytket/pytket/wasm/wasm.py

```diff
@@ -42,13 +48,15 @@ class WasmFileHandler:
         LANG_TYPE_EMPTY: None,
     }
 
-    def __init__(self, filepath: str, check_file: bool = True, int_size: int = 32):
```

Comment:
changing check_file to check is actually a breaking change and I probably should not have done this.

Reply:
ah it's actually still called `check_file` for WasmFileHandler so it could be fine? A bit confusing however